iset.mm

```diff
@@ -136318,6 +136318,118 @@ S C_ ( P ( ball ` D ) T ) ) $=
       WDXBWMCNYPYNWEYOWCWFWGWHWI $.
   $}
 
+  ${
+    limcflf.f $e |- ( ph -> F : A --> CC ) $.
+    limcflf.a $e |- ( ph -> A C_ CC ) $.
+    limcimo.b $e |- ( ph -> B e. CC ) $.
+    limcimo.bc $e |- ( ph -> B e. C ) $.
+    limcimo.bs $e |- ( ph -> B e. S ) $.
+    limcimo.c $e |- ( ph -> C e. ( K |`t S ) ) $.
+    limcimo.s $e |- ( ph -> S e. { RR , CC } ) $.
+    limcimo.ca $e |- ( ph -> { q e. C | q =//= B } C_ A ) $.
+    limcflfcntop.k $e |- K = ( MetOpen ` ( abs o. - ) ) $.
```

Comment:
I wonder whether you should define K as ` K = ( ( MetOpen `(abs o. -) ) |` t S ), since you want to look at the limit points of A either in RR or CC.  This would make C a redundant helper class, and bring you more in line with the original.

Reply:
I do appreciate these kinds of suggestions. There are a number of related theorems which are not intuitionizing easily. They include all the limit point theorems, perfect set theorems, and many of the places which do things like "look at what F does except at point B".

As for ` K `, I suppose in the next version of this I might not give a name to ` K ` at all and just write `` ( ( MetOpen ` (abs o. -) ) |` t S )`` itself, because it is lightly used in this proof.

I'm not sure I understand what would replace ` C ` though. ` C ` needs to contain ` B ` but ` A ` doesn't have to. Furthermore, ` C ` is open. I wonder if we should remove "limit point" from the comment for limcimo. That was copied from set.mm and I'm not sure how it applies here.